### ISSUE TITLE
Fix crashes

### ISSUE THREAD

**Issue by nils-keller-dev** [StarAppeal/generic-reddit-bot#3]

**Comment by StarAppeal** [comment]
Please restart the bots and pipe the output (console.log and console.error) into 2 different files, maybe there are some logs which are only logged into the console and not via Winston.
After you have done this, check the logs after the next crash occurs and check if there is helpful log.

**Comment by nils-keller-dev** [comment]
This is the log from 03.07.2022. I am already piping all logs (error and regular log) into a single `.txt`.  
Do you think it is necessary to separate the logfiles?

![IMG-20220703-WA0012.jpg](https://user-images.githubusercontent.com/60610467/235757726-f07c6439-f475-442b-915c-aac38ea211d2.jpg)

**Comment by StarAppeal** [comment]
I don't think it is neccessary to pipe the log into 2 seperate files, one is sufficient. Can you attach the .txt file onto this ticket?

**Comment by StarAppeal** [comment]
Wait, why is the log from the 03.07.2022? Don't we have something more recent?

**Comment by nils-keller-dev** [comment]
The txt gets overwritten with every restart xd  
But i can attach the log once it gets an error again. But I'm pretty sure the error would be the same as from a few months ago

**Comment by StarAppeal** [comment]
We had a few updates after the 03.07.2022 in the code, not that many though. Still want the most recent log.
